h2o-genmodel/build.gradle

```diff
@@ -27,6 +27,8 @@ dependencies {
   testCompile "junit:junit:${junitVersion}"
   compile "net.sf.opencsv:opencsv:2.3"
   compile "com.google.code.gson:gson:2.6.2"
+  compile "guru.nidi:graphviz-java:0.9.0"
```

Comment:
This dependency "weights" `172 Kb`.

Reply:
Excluding `guru.nidi.com.kitfox:svgSalamander:1.1.2` leads to 

```
java.lang.IllegalStateException: - Rasterizer explicitly set no null or
- neither Batik nor Salamander found on classpath.
	at guru.nidi.graphviz.engine.Renderer.toImage(Renderer.java:114)
	at guru.nidi.graphviz.engine.Renderer.toFile(Renderer.java:70)
	at guru.nidi.graphviz.engine.Renderer.lambda$toFile$3(Renderer.java:56)
	at guru.nidi.graphviz.engine.EngineResult.mapIO(EngineResult.java:63)
	at guru.nidi.graphviz.engine.Renderer.toFile(Renderer.java:54)
	at hex.genmodel.tools.PrintMojo.generateDotFromSource(PrintMojo.java:252)
	at hex.genmodel.tools.PrintMojo.handlePrintingToOutputFile(PrintMojo.java:220)
	at hex.genmodel.tools.PrintMojo.run(PrintMojo.java:209)
	at hex.genmodel.tools.PrintMojo.main(PrintMojo.java:46)
```

Reply:
I see. Thank you. Let's keep it in then. Still, the size is acceptable.